.github/workflows/CI.yml

```
@@ -40,6 +41,25 @@ jobs:
       - uses: actions/setup-python@v5
         with:
           python-version: 3.x
+      - name: Configure vendored OpenSSL for armv7
+        if: matrix.platform.target == 'armv7'
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y gcc-arm-linux-gnueabihf
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)